docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -376,7 +376,7 @@ Limitations:
     - XX% of each type of facility have CPAP or NICU capabilities
     - 
   * - 8
-    - XX relative risk for RDS mortality 
+    - XX relative risk for RDS mortality if facility has CPAP or NICU capabilities
```

Comment:
```suggestion
    - XX relative risk on RDS mortality of facility having CPAP or NICU capabilities vs. not
```
Not sure if this is the best wording, but trying to make it more precise. When I first read it, I was confused because I thought it meant "relative risk of RDS on mortality, if facility has CPAP or NICU capabilities".